### ISSUE TITLE
toggl auth fail

### ISSUE THREAD

**Issue by KUKARAF** [watercooler-labs/toggl-cli#72]
something seems to be wring with the auth command in the latest version. actual api key redacted.
Error occurs if built from sratch in main branch and from crates.io

command that fails: 

`toggl auth myy32api94key93429320432324923243`

![image](https://github.com/watercooler-labs/toggl-cli/assets/24418303/1b959fe1-55c7-4b6a-81e2-716a082cbb09)

**Comment by shantanuraj** [comment]
Yeah I seem to be able to reproduce this on my linux server.
Probably something wrong with the keyring package, I'll have a look and maybe add an alternative keystore.

Thanks for reporting!

**Comment by ThePitaGyros** [comment]
Same here. Got the same error message.

**Comment by evesdropper** [comment]
Have the same error message; I assume this hasn't been resolved?

**Comment by shantanuraj** [comment]
Hey @evesdropper yeah it hasn't been resolved yet.
I haven't been able to make much time to work on this lately.

I'll give it a go again today.

**Comment by shantanuraj** [comment]
I have a debug branch at [`shantanu/linux-keyring`](https://github.com/watercooler-labs/toggl-cli/tree/shantanu/linux-keyring)
with improved error logging.

This is the output of running `toggl auth` on a my ubuntu vps.

```bash
$ toggl auth API_KEY
Error writing to keyring: Couldn't access platform secure storage: SS error: result not returned from SS API
An error occurred when saving your credentials.
Make sure you are on the latest version of the app or file an issue here: https://github.com/watercooler-labs/toggl-cli/issues/new
```

**Comment by shantanuraj** [comment]
Hey all I've fixed the error in #78 

It's available in [v0.4.5](https://github.com/watercooler-labs/toggl-cli/releases/tag/v0.4.5)
https://crates.io/crates/toggl/0.4.5

Let me know if the error still persists for you.
Thanks for reporting.
